src/Otp.php

```diff
@@ -91,9 +91,20 @@ public function totp($secret, $timecounter = null)
     /* (non-PHPdoc)
      * @see Otp.OtpInterface::checkHotp()
     */
-    public function checkHotp($secret, $counter, $key)
+    public function checkHotp($secret, $counter, $key, $counterwindow = 5)
```

Comment:
I added the extra getter so that if this is used in some sort of REST API and the counter needs to be fetched I feel we should avoid sending it in the response.